### PR TITLE
Adding "Hide the used Port"-Option in HomebridgeStatusWidget

### PR DESCRIPTION
**Pull request by Staubgeborener**
This PR is a response to https://github.com/oznu/homebridge-config-ui-x/issues/558.

The used port is no longer displayed (on purpose) within the current version. The wrong spelling "running" was already fix'd. But I believe that it should be left open to the community whether they want to display the port or not. I mean, at least one person (also me) misses it. So i added an option "Hide the used Port" for the HomebridgeStatusWidget. You don't want the port? Check the box. You want the port? Uncheck it.

english_running
english_runningport
english_settings

I have also prepared all language files. But one or two lines have to be translated again. Not a big deal. 
P.S.: In this PR is also a corrected grammatical mistake in `de.json`.

### CODE REVIEW

**Comment by oznu** [comment]
Thanks; I'll check this out soon.

We can probably get the translations for "running on port" from an older commit: https://github.com/oznu/homebridge-config-ui-x/tree/140858974f48034ae64cb99aa3cc570fbef26d54/ui/src/i18n

**Comment by Staubgeborener** [comment]
Good idea. 
I used the old commit for the translation and also considered the recent changes (PR https://github.com/oznu/homebridge-config-ui-x/pull/573, PR https://github.com/oznu/homebridge-config-ui-x/pull/571 and PR https://github.com/oznu/homebridge-config-ui-x/pull/568). But there seems to be a conflict with ru.json.

**Comment by Caribsky** [comment]
Hi, @Staubgeborener 
Regarding ru.json.  All merges are correct.

File: ui/src/i18n/ru.json [hunk 1]
```diff
@@ -238,6 +239,7 @@
     "status.widget.label_systeminfo": "Информация о системе",
     "status.widget.label_unlock_layout": "Разблокировать макет",
     "status.widget.label_widget_has_no_options": "У этого виджета нет опций",
+    "status.widget.homebridgestatus.label_port": "Hide the used Port",
```
Comment (Caribsky) [inline ui/src/i18n/ru.json]: "status.widget.homebridgestatus.label_port": "Не показывать используемый порт",

**Comment by oznu** [comment]
Thanks @Staubgeborener and @Caribsky

**Pull request merged**